### ISSUE TITLE
no-unused-declaration false positive on values imported to wrap/inject

### ISSUE THREAD

**Issue by JDKardia** [cartant/tslint-etc#19]
Okay, so the simplest example to replicate is as follows. 

Say I have some file `bit.ts` that has some value or function that I export:
```tsx
const bit = 'just a bit';

export default bit;
```

then in the file next to it, `info-wrapper.ts`, I've got:
```tsx
import bit from './bit';

const infoWrapper = {
  bit
};

export default infoWrapper;
```

with this rule in my tslint.json:
```
...,
"no-unused-declaration": {
      "options": [{
        "declarations": false,
        "imports": true
      }],
      "severity": "error"
    },
...
```

TSLint will say that my import of `bit` is unused, even though it's being wrapped and exported.

**Comment by cartant** [comment]
Thanks for the detailed bug report.
